test/Microsoft.SqlTools.ServiceLayer.Test.Common/TestServerIdentity.cs

```diff
@@ -22,4 +23,10 @@ public enum TestServerType
         Azure,
         OnPrem
     }
+
+    public enum AuthenticationType
```

Comment:
no  we don't 

---
In reply to: [95272697](https://github.com/Microsoft/sqltoolsservice/pull/210#discussion_r95272697) [](ancestors = 95272697)